packages/turf-clusters/README.md

```diff
@@ -0,0 +1,172 @@
+# @turf/clusters
+
+# getCluster
+
+Get Cluster
+
+**Parameters**
+
+-   `geojson` **[FeatureCollection](http://geojson.org/geojson-spec.html#feature-collection-objects)** GeoJSON Features
+-   `filter` **Any** Filter used on GeoJSON properties to get Cluster
```

Comment:
Now that I think about it, only in `k-means` the number of clusters is known before applying the module (which often is considered its drawbacks), so you can easily "query" for a specific cluster `id` for example; on the other side if you apply `dbscan` or other algorithms you don't know a priori how many clusters will be identified.
It would be useful, I guess, to have a method that returns the total number of clusters in the `FeatureCollection` (basically how many groups of points have the same property with different values); it could be this one when passed `null` or `'all'` as filter.